assets/wizards/engagement/index.js

```diff
@@ -101,6 +113,15 @@ class EngagementWizard extends Component {
 				<HashRouter hashType="slash">
 					<Switch>
 						{ pluginRequirements }
+						<Route
```

Comment:
Nit: if `! newspack_engagement_wizard.has_reader_activation`, the route should not be rendered.

Reply:
Done in 7a9026fdd54b7ff5acfe2c8d572dce646e102873